src/successModal/index.tsx

```diff
@@ -1,100 +1,84 @@
-    color: ${color.white};
-  }
+import IllustratedSection from 'layout/section/illustratedSection'
 
-  & .kirk-modal--large {
-    text-align: center;
-  }
+import Modal from 'modal'
+import Button from 'button'
+import TextDisplay1 from 'typography/display1'
 
-  & .kirk-modal.kirk-successModal .kirk-modal-dialog {
+const StyledSuccessModal = styled(Modal)`
+  padding: 0;
+  text-align: center;
+  background-color: ${color.successBackground};
+
+  .kirk-modal-dialog {
+    display: flex;
+    justify-content: center;
+    padding: 0;
     margin: 0 auto;
-    max-width: ${modalSize.m};
-    height: 100%;
-    padding: ${space.xl} 0 ${footerHeight} 0;
+    width: 100%;
+    border-radius: 0;
     background-color: ${color.successBackground};
   }
 
-  /* Stronger selector to override Modal component styling */
-  &.kirk-modal-dimmer .kirk-modal.kirk-successModal .kirk-modal-dialog.kirk-modal-dialog {
-    width: auto;
-  }
-
-  & .kirk-modal--large.kirk-successModal .kirk-modal-dialog {
-    padding: ${space.xl} 0;
-  }
-
-  & .kirk-modal.kirk-successModal .kirk-modal-body {
-    padding: ${space.xl};
-    font-size: ${font.l.size};
-    line-height: ${font.l.lineHeight};
-    text-align: center;
-    align-items: center;
-  }
-
-  & .kirk-modal--large.kirk-successModal .kirk-modal-body {
-    position: relative;
+  .kirk-modal-body {
     display: flex;
-  }
-
-  & .kirk-modal--large .kirk-successModal-bodyItem {
-    width: 50%;
-  }
+    min-height: 100vh;
+    flex-direction: column;
 
-  & .kirk-modal--large .kirk-successModal-bodyItem:nth-child(odd) {
-    padding-right: ${space.xl};
-  }
-
-  & .kirk-modal--large .kirk-successModal-bodyItem:nth-child(even) {
-    padding-left: ${space.xl};
-  }
-
-  & .kirk-successModal-image {
-    width: 80%;
-    margin-bottom: ${space.xl};
+    @media (${responsiveBreakpoints.isMediaLarge}) {
+      flex-direction: row;
+      align-items: center;
+      flex-wrap: wrap;
+    }
   }
+`
 
-  & .kirk-modal--small .kirk-successModal-image {
-    max-height: 33vh;
-    object-fit: contain;
-  }
+const StyledIllustratedSection = styled(IllustratedSection)`
+  display: flex;
+  flex: 1;
 
-  & .kirk-modal--large .kirk-successModal-image {
-    margin-bottom: 0;
-  }
+  .kirk-illustratedSection-illustration {
+    padding-top: calc(2 * ${space.xxl});
+    justify-content: center;
+    align-items: center;
 
-  & .kirk-successModal-footer {
-    position: absolute;
-    bottom: 0;
-    left: 0;
-    width: 100%;
-    height: ${footerHeight};
-    padding: ${space.xl};
-    text-align: center;
+    @media (${responsiveBreakpoints.isMediaLarge}) {
+      padding-top: 0;
+    }
   }
 
-  & .kirk-modal--large .kirk-successModal-footer {
-    position: relative;
-    display: block;
-    margin: 0 auto;
+  .kirk-illustratedSection-content {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
   }
+`
+const SuccessTitle = styled(TextDisplay1)`
+  padding: ${space.xl};
+  flex: 1;
```

Comment:
Why the flex here ?
Why the specific title here ? could we reuse either a TheVoice or SubHeader ?

The global question is: why the content of the success modal is not a list of existing Kirk widgets and why do we need all these CSS overrides ?

Reply:
The CSS overrides were already there. I need _flex_ to control, well, _layout_. And sadly, kirk doest have no grid components.. 

`TheVoice` has its behavior on his own and I rather use Typography components to manage that to have a **pure** state. _Otherwise, it will be more and more overrides_

Should I replace to `TheVoice` then?